pkg/k8srbac/evaluator.go

```diff
@@ -45,39 +44,11 @@ func MakeClusterEvaluator(roles []*storage.K8SRole, bindings []*storage.K8SRoleB
 	// Collect cluster role bindings.
 	var clusterBindings []*storage.K8SRoleBinding
 	for _, binding := range bindings {
-		if binding.GetClusterRole() {
+		// A binding is only binding cluster-wide if cluster role is true _and_ namespace is empty. Namespaced
+		// bindings may bind to cluster roles as well.
+		if binding.GetClusterRole() && binding.GetNamespace() == "" {
```

Comment:
This condition appears 3 times how about extracting it to well named function in `pkg`?